cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
+  viewportWidth: 1080,
+  screenshotOnRunFailure: true,
+  reporter: 'mochawesome',
+  reporterOptions: {
+    reportDir: 'raw_reports',
```

Comment:
Ensure that the `raw_reports` directory exists or is correctly set up to store the test reports generated by `mochawesome`.